### PR TITLE
Port basic integration tests to Oak Functions Containers launcher

### PR DESCRIPTION
**Pull request by andrisaar**
We now run the basic end-to-end tests that we do with plain Oak Functions with Oak Functions on Oak Containers as well.

As always, this could use a ton of cleaning up, but... the tests are there, and more importantly, they pass!

Ref #4409

### CODE REVIEW

File: oak_functions_containers_launcher/src/main.rs [hunk 1]
```diff
@@ -39,8 +39,8 @@ async fn main() -> Result<(), anyhow::Error> {
         lookup_data_path: args.functions_args.lookup_data,
         // Hard-coded because we are not sure whether we want to configure the update interval.
         update_interval: Some(std::time::Duration::from_secs(60 * 10)),
-        // Fix the maximum size of a chunk to the proto limit size of 2 GiB.
-        max_chunk_size: ByteUnit::Gibibyte(2),
+        // gRPC messages are limited to 4 MiB.
+        max_chunk_size: ByteUnit::Mebibyte(4),
```
Comment (tiziano88) [inline oak_functions_containers_launcher/src/main.rs:43]: Just to check: if you increase this limit by 1 byte, does it actually start failing?

File: xtask/src/containers.rs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                crate::testing::run_step(build_kernel()),
+                crate::testing::run_step(build_system_image()),
+                crate::testing::run_step(build_oak_functions_bundle()),
+                crate::testing::run_step(build_oak_functions_launcher())
+            );
+        })
+        .await;
+}
+
+pub fn run_oak_functions_launcher_example_with_lookup_data(
```
Comment (tiziano88) [inline xtask/src/containers.rs:70]: I wonder if we should move some of this dependency management in `just`, now that we are more familiar with it. We can still spawn out to call `just` from Rust tests, but at least we can declare the dependency tree in the `justfile` itself. Anyways, not sure it's worth doing at the moment, but we can chat about it separately.

**Review by tiziano88 — APPROVED** [review]

File: oak_functions_containers_launcher/src/main.rs (hunk 1 above)
Comment (andrisaar) [reply oak_functions_containers_launcher/src/main.rs:43]: I haven't checked it by 1 byte, but 8 MB was too much (the failing error said that max is 4 MB).

File: xtask/src/containers.rs (hunk 2 above)
Comment (pmcgrath17) [reply xtask/src/containers.rs:70]: Would this involve using bazel for rust? Has this idea been entertained by Oak before?

Comment (andrisaar) [reply xtask/src/containers.rs:78]: We have thought about it to the extent of "someone should play around with it to see if it is feasible", but it hasn't ever been a priority for us.

**Pull request merged**